### ISSUE TITLE
SublimeLinter not working on Sublime Text build 4107

### ISSUE THREAD

**Issue by Yofret-Rios** [SublimeLinter/SublimeLinter#1814]
Hi guys, first and foremost thanks for this awesome that has serve me well for a lot years.

I couldn't find a similar issue, so I decided to create a new one, please let me know if there is an existing issue, so this one can be deleted,.

I've recently updated Sublime Text to Build 4107, and I notice that the Linter Stop working for some reason

here are the logs I'm receiving from the console:
```
SublimeLinter: WARNING: 'index.jsx' puts 15 eslint tasks on the queue.
SublimeLinter: WARNING: 'index.jsx' puts in total 15(!) tasks on the queue.
SublimeLinter: WARNING: 'index.jsx' puts 15 eslint tasks on the queue.
SublimeLinter: WARNING: 'index.jsx' puts in total 15(!) tasks on the queue.
SublimeLinter: WARNING: 'index.jsx' puts 15 eslint tasks on the queue.
SublimeLinter: WARNING: 'index.jsx' puts in total 15(!) tasks on the queue.
SublimeLinter: WARNING: 'index.jsx' puts 15 eslint tasks on the queue.
SublimeLinter: WARNING: 'index.jsx' puts in total 15(!) tasks on the queue.
```

This happens happens every time I try to lint a particular view,

here is my configuration file, manually specified the node path
```
// SublimeLinter Settings - User
{
  "lint_mode": "load_save",
  "linters": {
    "eslint": {
      "env": { 
        "PATH":"../.nvm/versions/node/v14.16.1/bin/" 
      }
    }
  }
}
```

please let me know if there guidance I could follow, thanks a bunch!

**Comment by kaste** [comment]
It's tracked at https://github.com/SublimeLinter/SublimeLinter-eslint/issues/310

Let's fix it.

**Comment by kaste** [comment]
Fix released as 4.2.2
